github-sync/github-data/sigstore/repositories.yaml

```diff
@@ -1410,6 +1410,67 @@ repositories:
         dismissalRestrictions:
           - tuf-root-signing-codeowners
           - sigstore-keyholders
+  - name: root-signing-staging
+    owner: sigstore
+    description: "Staging TUF repository for Sigstore trust root"
+    homepageUrl: ""
+    defaultBranch: main
+    allowAutoMerge: false
+    allowMergeCommit: true
+    allowRebaseMerge: false
+    allowSquashMerge: false
+    archived: false
+    autoInit: false
+    deleteBranchOnMerge: false
```

Comment:
Does automation need a static branch, or does it create new branches frequently? Might want this on.

Reply:
Not that frequently, but a branch per signing event. I would maybe rather start with not deleting branches for easier forensics